package.json

```diff
@@ -42,12 +42,12 @@
     "karma-rollup-preprocessor": "^5.1.1",
     "mocha": "^5.0.0",
     "prettier": "^1.10.2",
-    "rollup": "^0.55.1",
+    "rollup": "^0.68.2",
     "rollup-plugin-babel": "^3.0.3",
     "rollup-plugin-commonjs": "^8.3.0",
     "rollup-plugin-jscc": "^0.3.3",
     "rollup-plugin-node-resolve": "^3.0.2",
-    "rollup-plugin-typescript": "^0.8.1",
+    "rollup-plugin-typescript2": "^0.19.2",
```

Comment:
Since [rollup-plugin-typescript](https://github.com/rollup/rollup-plugin-typescript) seems to be unable to generate type definitions, I used [rollup-plugin-typescript2](https://github.com/ezolenko/rollup-plugin-typescript2).
https://github.com/rollup/rollup-plugin-typescript/issues/54